src/view_control.cpp

```diff
@@ -250,6 +250,9 @@ void View::DrawTimeSpanningElement(DeviceContext *dc, Object *element, System *s
         x2 = last->GetDrawingX() + last->GetRightBarLineXRel();
         spanningType = SPANNING_MIDDLE;
     }
+    // Return otherwise: this should only happen if the time spanning element is encoded in the wrong measure
```

Comment:
It would be good to add a warning, however maybe be better placed when resolving references in Doc::PrepareDarwing. Placing control (time spanning) events not in the measure where they start is a very bad encoding practice.

You should add `{}` anyway.